### ISSUE TITLE
Unable to install on Expo Bare project (iOS)

### ISSUE THREAD

**Issue by vinjegaard** [lottie-react-native/lottie-react-native#565]
### Description

Get error message when running `pod install`
```
The following Swift pods cannot yet be integrated as static libraries:

The Swift pod `lottie-react-native` depends upon `React`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.
```
### Steps to Reproduce

1. `expo init LottieTest` (select "Bare Workflow - minimal" and follow wizard)
2. `cd LottieTest`
3. `npm i --save lottie-react-native@3.0.2` (side note on npm RN v59 install guide does not include @3.0.2)
4. `npm i --save lottie-ios@3.0.3`
5. `react-native link lottie-ios`
6. `react-native link lottie-react-native`
7. `cd ios`
8. `pod install`

I've also tried creating a dummy swift file and added the binding header. If I add `use_modular_headers!` to podfile I'm unable to build the project because of glog and other warnings.

**Comment by Uniphix** [comment]
Not sure if this is related to the same issue when I do pod install with Expo SDK 35 / React Native 0.59

`[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In Podfile:
    ExpoKit/Core (from `http://github.com/expo/expo.git`, tag `ios/2.13.0`) was resolved to 35.0.0, which depends on
      lottie-ios (~> 2.5.0)

    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 3.0.2, which depends on
      lottie-ios (~> 3.0.3)`

**Comment by allpwrfulroot** [comment]
Quick tip: lock lottie-react-native to v2.6.1 for expokit apps at the moment.
Follow thread here: https://github.com/expo/expo/issues/5559
